examples/mt-pytorch/client.py

```diff
@@ -35,8 +36,16 @@ def evaluate(self, parameters, config):
 
 
 # Start Flower client
-fl.client.start_numpy_client(
-    server_address="0.0.0.0:9092",
-    client=FlowerClient(),
-    transport="grpc-rere",
-)
+if restful_mode:
+    fl.client.start_numpy_client(
+        server_address="http://0.0.0.0:9093",
+        client=FlowerClient(),
+        rest=True,
+        transport="rest",
+    )
+else:
+    fl.client.start_numpy_client(
+        server_address="0.0.0.0:9092",
+        client=FlowerClient(),
+        transport="grpc-rere",
+    )
```

Comment:
`rest=True` is deprecated and will be removed. How about just adding comments to make it easier to switch between gRPC-rere and REST?
```suggestion
    fl.client.start_numpy_client(
        server_address="0.0.0.0:9092",  # "0.0.0.0:9093" for REST
        client=FlowerClient(),
        transport="grpc-rere",  # "rest" for REST
    )
```